docs/contribute/code.md

```diff
@@ -136,6 +147,23 @@ If your contribution focuses on a bugfix and you want it featured in the next st
 - [Sync a fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks/syncing-a-fork)
 - [Merge an upstream repository into your fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks/merging-an-upstream-repository-into-your-fork)
 
+### Reproducing job failures
+
+If one of your CI jobs fails, it will print a message explaining how to reproduce the task locally. Typically that involves running the task against the right template:
+
+```shell
+yarn task --task e2e-tests --template=cra-default-ts --start-from=install
```

Comment:
Not sure if you forgot to change this to `react-vite/default-ts`, but if cra was intended, then it should have a slash

```suggestion
yarn task --task e2e-tests --template=cra/default-ts --start-from=install
```

Reply:
Feature request: probably a good idea to also accept fully dashed template names, e.g. `react-vite-default-ts`, as it's how the folder names inside of `sandbox` are created, thus it's easier for contributors to understand how to find the template names.

This might not be 100% necessary though it'd be nice, however if users supply a wrong template they do get a nice feedback with the right options:
<img width="660" alt="image" src="https://user-images.githubusercontent.com/1671563/194707057-df82c9ca-9a1d-4ad9-9932-9e30b5aee89b.png">

Reply:
Yeah, there is an idea to allow either `--template=react-vite/default-js` OR `--sandbox=react-vite-default-js` (or both, in which case the `sandbox` step would create the project in the directory specified. I decided to leave that off this PR for now as it isn't really needed yet.